.vib/codeigniter/goss/goss.yaml

```diff
@@ -0,0 +1,12 @@
+gossfile:
+  # Goss tests exclusive to the current container
+  ../../codeigniter/goss/codeigniter.yaml: {}
+  # Goss tests exclusive to the current container
+  # Load scripts from .vib/common/goss/templates
+  ../../common/goss/templates/check-app-version.yaml: {}
+  ../../common/goss/templates/check-broken-symlinks.yaml: {}
+  ../../common/goss/templates/check-ca-certs.yaml: {}
+  ../../common/goss/templates/check-binaries.yaml: {}
+  ../../common/goss/templates/check-directories.yaml: {}
+  ../../common/goss/templates/check-linked-libraries.yaml: {}
+  ../../common/goss/templates/check-sed-in-place.yaml: {}
```

Comment:
Could you please update your from master and add here the new genera test `check-spdx` added at  https://github.com/bitnami/containers/pull/26723? I would be great if you could add it to each of your opened PRs, thanks.

Reply:
I will add the check-spdx for all PR's at once on a later moment